### PR TITLE
Bring back validation groups

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | yes |
| BC breaks?      | no |
| Related tickets | - |
| License         | MIT |

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
@pamil Could you also quickly update the Customization Guide, so that we can close this problem in 1 step? :)

File: docs/customization/validation.rst [hunk 1]
```diff
@@ -37,15 +37,12 @@ Give it a new, custom validation group - ``[app_product]``.
 
     When using custom validation messages see `here how to add them <http://symfony.com/doc/current/validation/translations.html>`_.
 
-2. Configure the new validation group in the ``app/config/config.yml``.
+2. Configure the new validation group in the ``app/config/services.yml`` by sett.
```
Comment (michalmarcinkowski) [inline docs/customization/validation.rst]: `sett`?

Comment (TheMadeleine) [inline docs/customization/validation.rst]: `by sett`? why did you add this part?

Comment (NeverResponse) [inline docs/customization/validation.rst]: My guess is, Sett is somehow a coauthor.

Comment (pamil) [inline docs/customization/validation.rst]: https://en.wikipedia.org/wiki/Submarine_escape_training_facility

**Comment by pjedrzejewski** [comment]
Requires rebasing.

**Comment by pamil** [comment]
@pjedrzejewski rebased.

**Comment by pjedrzejewski** [comment]
Thanks Kamil!
